server/sockets/list.js

```diff
@@ -477,6 +477,102 @@ const changeListType = (socket, dashboardClients, cohortClients, listClients) =>
       });
   });
 
+const emitListsOnRemoveCohortMember = (socket, dashboardClients, listClients) =>
+  socket.on(CohortActionTypes.REMOVE_MEMBER_SUCCESS, data => {
+    const { cohortId, userId } = data;
+
+    List.find({ cohortId })
+      .populate('cohortId', 'memberIds')
+      .lean()
+      .exec()
+      .then(docs => {
+        if (docs) {
+          const {
+            cohortId: { memberIds: cohortMemberIds }
+          } = docs[0];
+          const listIdsUserWasRemovedFrom = [];
+          const listIdsUserRemained = [];
+
+          docs.forEach(doc => {
+            const { type, viewersIds } = doc;
+            const isListViewer = checkIfArrayContainsUserId(viewersIds, userId);
+            const listId = doc._id.toString();
+
+            if (isListViewer) {
+              listIdsUserRemained.push(listId);
+            } else if (!isListViewer && type === ListType.SHARED) {
+              listIdsUserWasRemovedFrom.push(listId);
+            }
+          });
+
+          if (listIdsUserWasRemovedFrom.length > 0) {
+            if (dashboardClients.has(userId)) {
+              const { socketId } = dashboardClients.get(userId);
+
+              socket.broadcast
+                .to(socketId)
+                .emit(ListActionTypes.REMOVE_BY_IDS, listIdsUserWasRemovedFrom);
+            }
+
+            listIdsUserWasRemovedFrom.forEach(listId => {
+              socket.broadcast
+                .to(`sack-${listId}`)
```

Comment:
Consider using channel url helpers like `listChannel = listId =>`. Please also use consistent naming, even if it differs from the display ones.